.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 15
```

Comment:
I would be more conservative at the beginning.
I think this is a better setting:
```suggestion
daysUntilStale: 365
# Number of days of inactivity before a stale issue is closed
daysUntilClose: 30
```